shared/js/background/startup.es6.js

```diff
@@ -12,11 +12,9 @@ const tdsStorage = require('./storage/tds.es6')
 const trackers = require('./trackers.es6')
 const dnrSessionId = require('./dnr-session-rule-id')
 const { fetchAlias, showContextMenuAction } = require('./email-utils.es6')
-const manifestVersion = browserWrapper.getManifestVersion()
-/** @module */
+const { startupReadyResolve } = require('./ready.es6')
```

Comment:
We can drop the .es6 from this file; it's only used when transpilation is needed which isn't the case ever.